src/openforms/prefill/contrib/stufbg/plugin.py

```diff
@@ -143,7 +143,18 @@ def get_identifier_value(
             return submission.auth_info.value
 
         if identifier_role == IdentifierRoles.authorizee:
-            return submission.auth_info.machtigen.get("identifier_value")
+            legacy_fallback = submission.auth_info.machtigen.get("identifier_value")
```

Comment:
I see the code is duplicated but I guess we want that in this explicit way.  I would have this in one place to be honest, in contrib for example.

Reply:
IMO that breaks the isolation of plugins and ties them too much to generic open-forms code. in 3.0 we will probably remove the fallback behaviour and then it's just a similar-but-not-the-same pattern. maybe it could become a method on `AuthInfo` at some point, but the extra checks if it's a BSN/KVK kind of make it plugin-specific what needs to be checked.

In essence this is one-time only code, but because StUF-BG and HaalCentraal BRP plugins are alternatives for the same functionality, this code is duplicated.